modules/openapi-generator/src/main/resources/Java/libraries/rest-assured/build.gradle.mustache

```diff
@@ -96,17 +96,16 @@ if(hasProperty('target') && target == 'android') {
 
 ext {
     swagger_annotations_version = "1.5.21"
-    rest_assured_version = "3.1.0"
+    rest_assured_version = "3.3.0"
     junit_version = "4.12"
-    gson_version = "2.6.1"
-    gson_fire_version = "1.8.2"
+    gson_version = "2.8.5"
+    gson_fire_version = "1.8.3"
 {{#joda}}
     jodatime_version = "2.9.9"
 {{/joda}}
 {{#threetenbp}}
-    threetenbp_version = "1.3.5"
+    threetenbp_version = "1.3.8"
 {{/threetenbp}}
-    okio_version = "1.13.0"
```

Comment:
Seems like this one should be kept as I now get the following error:
```
➜  rest-assured git:(viclovsky-rest_assured_update_dependencies) gradle test
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/williamcheng/Code/openapi-generator/samples/client/petstore/java/rest-assured/build.gradle' line: 111

* What went wrong:
A problem occurred evaluating root project 'petstore-rest-assured'.
> Could not get unknown property 'okio_version' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 12s
```

Reply:
William , you are right. I have deleted it by mistake. I'll return it back: okio_version = "1.13.0"